phpipam/request/request.go

```diff
@@ -147,15 +147,21 @@ func (r *Request) Send() error {
 
 	switch r.Method {
 	case "OPTIONS", "GET", "POST", "PUT", "PATCH", "DELETE":
-		bs, err := json.Marshal(r.Input)
-		log.Debugf("Request Body Debug ................... %s", bs)
-		if err != nil {
-			return fmt.Errorf("Error preparing request data: %s", err)
+		if r.Input != nil {
+			bs, err := json.Marshal(r.Input)
+			log.Debugf("Request Body Debug ................... %s", bs)
+			if err != nil {
+				return fmt.Errorf("Error preparing request data: %s", err)
+			}
+			buf := bytes.NewBuffer(bs)
+			log.Debugf("Request URL Debug ...................Method: %s, UR: %s/%s%s", r.Method, r.Session.Config.Endpoint, r.Session.Config.AppID, r.URI)
```

Comment:
Please remove this line.
The same log is below.

Reply:
It's done !